airflow/models/dagrun.py

```diff
@@ -65,14 +66,15 @@ class DagRun(Base, LoggingMixin):
     )
 
     def __init__(self, dag_id=None, run_id=None, execution_date=None, start_date=None, external_trigger=None,
-                 conf=None, state=None):
+                 conf=None, state=None, run_type=None):
         self.dag_id = dag_id
         self.run_id = run_id
         self.execution_date = execution_date
         self.start_date = start_date
         self.external_trigger = external_trigger
         self.conf = conf
         self.state = state
+        self.run_type = run_type
```

Comment:
Probably it's worth to use this column in index

Reply:
Yeah, we should look at how we query dag_run table, as I'd guess we want a multi-column index on run_type + something else (exeuction date? dag id?) as we don't (I think) ever select just by run_type?

Reply:
@turbaszek What indexes did we decide on?

Reply:
I think we should keep the one we have. The DagRunType is only used in query only in `dag_list_dag_runs`.

Reply:
https://github.com/apache/airflow/blob/bae5cc2f5ca32e0f61c3b92008fbd484184448ef/airflow/jobs/scheduler_job.py#L573 Can you look at it?

Reply:
The scheduler (well dag parsing process) looks at DagRuns where type is not Backfill.

Reply:
https://github.com/apache/airflow/blob/bae5cc2f5ca32e0f61c3b92008fbd484184448ef/airflow/jobs/scheduler_job.py#L1158-L1188

Reply:
(As Kamil already commented. Missed that comment somehow)